datafusion/substrait/src/lib.rs

```diff
@@ -15,8 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
-pub mod consumer;
-pub mod producer;
+pub mod logical_plan;
```

Comment:
I like these name changes. Make it much more clear what the library is trying to offer.